### ISSUE TITLE
setup.py should also update vistrails branch if needed

### ISSUE THREAD

**Issue by doutriaux1** [UV-CDAT/uvcmetrics#101]
sometimes bug fixes are in vistrails branch not in this repo.

**Comment by isubsmith** [comment]
I *think* this ended up being a uvcmetrics issue. The problem is setup.py in uvcmetrics doesn't necessarily delete/replace/update all of the files in {uvcdat directory}site-packages/metrics. It's fine if you build uvcdat and get a good version of metrics; it isn't fine if you are a developer working with uvcmetrics.



________________________________

**Comment by doutriaux1** [comment]
Yes but @painter1 closed it. The title wasn't specific so I created a new one just for this
